lib/zipkin/span.rb

```diff
@@ -45,15 +46,30 @@ def get_baggage_item(key)
       nil
     end
 
-    # Add a log entry to this span
+    # @deprecated Use {#log_kv} instead.
+    # Reason: event is an optional standard log field defined in spec and not required.  Also,
+    # method name {#log_kv} is more consistent with other language implementations such as Python and Go.
     #
+    # Add a log entry to this span
     # @param event [String] event name for the log
     # @param timestamp [Time] time of the log
     # @param fields [Hash] Additional information to log
     def log(event: nil, timestamp: Time.now, **fields)
+      warn "Span#log is deprecated.  Please use Span#log_kv instead."
       nil
     end
 
+    # Add a log entry to this span
+    # @param timestamp [Time] time of the log
+    # @param fields [Hash] Additional information to log
+    def log_kv(timestamp: Time.now, **fields)
+      @logs.push({
```

Comment:
how would this be used?

Reply:
https://github.com/openzipkin/zipkin-go-opentracing/blob/4c9fbcbd6d73a644fd17214fe475296780c68fb5/zipkin-recorder.go#L179-L185 This is what I found for go implementation but I'm not entirely sure what all of this does.

Reply:
@indrekj Thank you your comment.
I'm trying to use this library for our service tracing with event logging, however it doesn't implement .
I found an implementation in `zipkin-go-opentracing` too.
It seems to send logs that include `timestamp` and `fields` converted to strings in Annotations.
I could not find any other implementation, but since it is the official implementation of openzipkin I tried a similar implementation.